src/components/File/tabs/associations/AssociationsTab.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import { EMPTY_ARRAY, EMPTY_OBJECT, NS_FILE_UNITS } from '../../../../helpers/consts';
+import { actions, selectors } from '../../../../redux/modules/lists';
+import { actions as fileActions } from '../../../../redux/modules/files';
+import { selectors as unitsSelectors } from '../../../../redux/modules/content_units';
+import * as shapes from '../../../shapes';
+import NewAssociations from './NewAssociations';
+
+class AssociationsTab extends Component {
+
+  static propTypes = {
+    file: shapes.File,
+    units: PropTypes.arrayOf(shapes.ContentUnit),
```

Comment:
Should be names 'items' ?

Reply:
I use this names because of in the association CU to content have same names. I want to rename as here as there in special branch.